rpc/params/applications.go

```diff
@@ -312,6 +318,16 @@ type DestroyApplicationParams struct {
 	MaxWait *time.Duration `json:"max-wait,omitempty"`
 }
 
+// DestroyApplicationParams holds parameters for the
+// Application.DestroyApplication call.
+type DestroyApplicationParams struct {
+	DestroyApplicationParamsV15
```

Comment:
Don't embed this. Just copy the fields from the old version so that it remains flat.